content/docs/faq.mdx

```diff
@@ -62,4 +62,216 @@ description: |
+  >
+  This is caused by missing KVM support.
+  This may be the case if:
+
+    * you are running an ARM64 image on x86_64 (or viceversa)
+    * you are using a virtual machine for development / deployment
+    * you are using Windows (with WSL) or macOS
+    * your system simply doesn't have KVM support.
+
+  If you are using `kraft run`, use the `-W` option to disable KVM support requirement.
+  If you are using QEMU, be sure to **not** use the `-enable-kvm` nor the `-accel kvm` options.
+  Firecracker currently requires KVM support;
+  you can't run it on a KVM-less system.
+  </FaqItem>
+  <FaqItem
+    question="How do I stop a running QEMU instance?"
+  >
+  Use `Ctrl+a x` (i.e. press `Ctrl` and `a` keys simultanously, release, then press `x` key) to close an open QEMU instance in the terminal.
+  You can also kill all QEMU instances by running, in another console:
+
+  ```console
+  pkill -f qemu-system
+  ```
+  </FaqItem>
+  <FaqItem
+    question="How do I stop a running Firecracker instance?"
+  >
+  There's no keyboard shortcut to stop a running Firecracker instance.
+  You either find out the pid and kill it:
+
+  ```console
+  pgrep -f firecracker
+  kill <PID>
+  ```
+
+  Replace `<PID>` with the actual PID (_Process ID_) returned by `pgrep`.
+
+  You can also kill all Firecracker instances by running, in another console:
+
+  ```console
+  pkill -f firecracker
+  ```
+  </FaqItem>
+  <FaqItem
+    question="How do I stop a running KraftKit instance (started with `kraft run`)?"
+  >
+  If you use `Ctrl+c` that will close the current KraftKit controlling process, but may not fully remove the runtime information, including potential port mappings and potential started QEMU / Firecracker / Xen processes.
+
+  Use `--rm` as argument to `kraft run` to remove the underlying runtime when using `Ctrl+c`.
+
+  To list all active / inactive KraftKit instances, use:
+
+  ```console
+  kraft ps -a
+  ```
+
+  To stop an instance or all instances, or to remove an instance or all instances use:
+
+  ```console
+  kraft stop <instance_name>
+  kraft stop --all
+  kraft rm <instance_name>
+  kraft rm --all
+  ```
+  </FaqItem>
+  <FaqItem
+    question="I get the error `port 0.0.0.0:8080 is already in use by ...`
+    What do I do?"
+  >
+  There is another instance running that is using port `8080`.
+  That may be a KraftKit instance, a QEMU instance, a Firecracker instance, or something else.
+
+  First, check the listening services to detect what is using the port:
+
+  ```console
+  netstat -tlpn
+  ```
+
+  You can also check running KraftKit or QEMU or Firecracker instances:
+
+  ```
+  kraft ps -a
+  ps -ef | grep qemu-system
+  ps -ef | grep firecracker
+  ```
+
+  You can stop them to free the port:
+
+  ```console
+  kraft rm <instance_name>
+  kill <qemu_PID>
+  kill <firecracker_PID>
+  ```
+
+  You can also stop all running instances:
+
+  ```console
+  kraft rm --all
+  pkill -f qemu-system
+  pkill -f firecracker
+  ```
```

Comment:
Maybe also add a `fuser -k port/tcp` in case the port is not being used by a qemu/firecracker process.